packages/phone-number-privacy/src/models/account.ts

```diff
@@ -0,0 +1,22 @@
+import { Model } from './model'
+
+export const ACCOUNTS_TABLE = 'accounts'
+export enum ACCOUNTS_COLUMNS {
+  address = 'address',
+  createdAt = 'created_at',
+  numLookups = 'num_lookups',
+  didMatchmaking = 'did_matchmaking',
+}
+
+export class Account extends Model {
+  address: string
```

Comment:
Can we store user-defiend types? There exists an "Address" object already defined in the contrackit library that we can use.